app/controllers/pages_controller.rb

```diff
@@ -1,9 +1,17 @@
 # frozen_string_literal: true
 
 class PagesController < SecureController
+  def create
+    p = Page.create!(params.require(:data).require(:attributes)
+                       .permit([:filename, :structure, :resource_id, :position]))
```

Comment:
I have Rubocop configured in this project, but it doesn't seem to care about the indentation here.  I have my continuation indent set to 2, so that's why it's indented two spaces.